lib/datasource/remote/model/region_model.dart

```diff
@@ -0,0 +1,47 @@
+class RegionModel {
+  final String id;
+  final String founder;
+  final String status;
+  final String description;
+  final String locationJson;
```

Comment:
what is this? I see no way to add this when creating a region?

Reply:
oh i see its just 
 'locationJson': '{lat:$latitude,lon:$longitude}',
          'latitude': latitude,
          'longitude': longitude

Reply:
location json is somewhat undefined

Maybe we want to have a more generic metadata field too?